### ISSUE TITLE
Add Translations

### ISSUE THREAD

**Issue by BrooklinJazz** [BrooklinJazz/elixir_newbie_website_v2#28]
Look into translating the main text on Elixir Newbie. We could start with Spanish since we have several Spanish-speaking contributors.

Once we have that going, we could look into options for translating blogs/DYA articles.

**Comment by IciaCarroBarallobre** [comment]
The idea is used gettext to do it? :)

**Comment by BrooklinJazz** [comment]
Exactly! The same way that you have done it in your capstone. We might encounter some extra difficulties since we're using Markdown blogs with nimble publisher though 🤔

**Comment by IciaCarroBarallobre** [comment]
Oh, including the set_locale library? Or do you prefer a language toggle/selection or some thing like that? Or maybe both 😊

https://github.com/smeevil/set_locale

**Comment by BrooklinJazz** [comment]
I trust your judgment and want to encourage you to follow your creativity!

If you're looking for input, then "both" seems pretty good!
